### ISSUE TITLE
Show external link and register here fields

### ISSUE THREAD

**Issue by dwaxweiler** [wordpress-connector-for-mobilizon/connector-mobilizon#21]
> Would also be nice if the plugin could show the value for the 'external (ticket) link' and 'register here' fields.

by @0spinboson

**Comment by dwaxweiler** [comment]
I imagine it is the `externalParticipationUrl` field we are talking about: https://framagit.org/framasoft/mobilizon/-/blob/4.1.0/schema.graphql?ref_type=tags#L2847

**Comment by 0spinboson** [comment]
yeah ideally the link would either go to the external registration page or just to the event page on the mobilizon instance so people can register/login or subscribe anonymously as a participant.
